src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentTableColumns.java

```diff
@@ -12,11 +12,12 @@
     SAMPLE("Sample"), CONTIG("Chromosome"), START("Start"), END("End"),
     NUM_PROBES("Num_Probes"), MEAN("Segment_Mean"), SD("Segment_Std"), CALL("Segment_Call"),
     NUM_TARGETS("Num_Targets"), NUM_SNPS("Num_SNPs"),
-    SEGMENT_MEAN_POSTERIOR_MEAN("Segment_Mean_Post_Mean"),
-    SEGMENT_MEAN_POSTERIOR_STANDARD_DEVIATION("Segment_Mean_Post_Std"),
-    MINOR_ALLELE_FRACTION_POSTERIOR_MEAN("MAF_Post_Mean"),
-    MINOR_ALLELE_FRACTION_POSTERIOR_STANDARD_DEVIATION("MAF_Post_Std"),
-
+    SEGMENT_MEAN_POSTERIOR_MODE("Segment_Mean_Post_Mode"),
```

Comment:
hmm, in this case, ACNV specifically sets `PosteriorSummary.center()` to be the mode, so i think it's worth keeping this extra information in the column header.  in any case, if we change this in the future, the code will have to change in two places regardless---the place where we set the center value, and the place where we document what that center is---so why not let the buck stop at the column header?

however, at the same time, we should keep `PosteriorSummary` flexible enough to store whatever we want in the `center` field.